### ISSUE TITLE
lld 14.0.6: test suite is failing

### ISSUE THREAD

**Issue by kloczek** [llvm/llvm-project#56222]
lld build from its own dist standalone tar ball.
cmake setup:
```console
[tkloczko@devel-g2v x86_64-redhat-linux-gnu]$ cmake -L
CMake Warning:
  No source or binary directory provided.  Both will be assumed to be the
  same as the current working directory, but note that this warning will
  become a fatal error in future CMake releases.


CMake Error: The source directory "/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
-- Cache values
CMAKE_BUILD_TYPE:STRING=RelWithDebInfo
CMAKE_CXX_ARCHIVE_CREATE:STRING=<CMAKE_AR> Dqc <TARGET> <LINK_FLAGS> <OBJECTS>
CMAKE_CXX_ARCHIVE_FINISH:STRING=<CMAKE_RANLIB> -D <TARGET>
CMAKE_C_ARCHIVE_CREATE:STRING=<CMAKE_AR> Dqc <TARGET> <LINK_FLAGS> <OBJECTS>
CMAKE_C_ARCHIVE_FINISH:STRING=<CMAKE_RANLIB> -D <TARGET>
CMAKE_INSTALL_PREFIX:PATH=/usr
LLD_BUILD_TOOLS:BOOL=ON
LLD_DEFAULT_LD_LLD_IS_MINGW:BOOL=OFF
LLD_USE_VTUNE:BOOL=OFF
LLD_VENDOR:STRING=
LLVM_BINARY_DIR:PATH=
LLVM_DIR:PATH=/usr/lib64/cmake/llvm
LLVM_ENABLE_IDE:BOOL=OFF
LLVM_ENABLE_LTO:STRING=OFF
LLVM_EXPORT_SYMBOLS_FOR_PLUGINS:BOOL=OFF
LLVM_EXTERNAL_LIT:STRING=/usr/bin/lit
LLVM_FORCE_ENABLE_STATS:BOOL=OFF
LLVM_LIT:FILEPATH=/usr/bin/lit
LLVM_LIT_ARGS:STRING=-sv
LLVM_MAIN_INCLUDE_DIR:PATH=/usr/include
LLVM_MAIN_SRC_DIR:PATH=/usr/share/llvm/src
LLVM_PARALLEL_COMPILE_JOBS:STRING=
LLVM_PARALLEL_LINK_JOBS:STRING=
LLVM_PTHREAD_LIBRARY_PATH:FILEPATH=/usr/lib64/libpthread.a
LLVM_SOURCE_PREFIX:STRING=
LLVM_TABLEGEN_EXE:FILEPATH=/usr/bin/llvm-tblgen
LLVM_USE_RELATIVE_PATHS_IN_DEBUG_INFO:BOOL=OFF
LLVM_USE_RELATIVE_PATHS_IN_FILES:BOOL=OFF
```
And test suite is failing:
```console
+ cd lld-14.0.6.src
+ /usr/bin/make -O -j48 V=1 VERBOSE=1 -C x86_64-redhat-linux-gnu test ARGS=--output-on-failure
make: Nothing to be done for 'test'.
+ /usr/bin/cmake --build x86_64-redhat-linux-gnu -j48 --verbose --target check-lld
/usr/bin/cmake -S/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src -B/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/gmake  -f CMakeFiles/Makefile2 check-lld
gmake[1]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/cmake -S/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src -B/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/CMakeFiles 88
/usr/bin/gmake  -f CMakeFiles/Makefile2 test/CMakeFiles/check-lld.dir/all
gmake[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f wasm/CMakeFiles/WasmOptionsTableGen.dir/build.make wasm/CMakeFiles/WasmOptionsTableGen.dir/depend
/usr/bin/gmake  -f CMakeFiles/intrinsics_gen.dir/build.make CMakeFiles/intrinsics_gen.dir/depend
/usr/bin/gmake  -f COFF/CMakeFiles/COFFOptionsTableGen.dir/build.make COFF/CMakeFiles/COFFOptionsTableGen.dir/depend
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/wasm /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/wasm /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/wasm/CMakeFiles/WasmOptionsTableGen.dir/DependInfo.cmake --color=
/usr/bin/gmake  -f ELF/CMakeFiles/ELFOptionsTableGen.dir/build.make ELF/CMakeFiles/ELFOptionsTableGen.dir/depend
/usr/bin/gmake  -f MachO/CMakeFiles/MachOOptionsTableGen.dir/build.make MachO/CMakeFiles/MachOOptionsTableGen.dir/depend
/usr/bin/gmake  -f MinGW/CMakeFiles/MinGWOptionsTableGen.dir/build.make MinGW/CMakeFiles/MinGWOptionsTableGen.dir/depend
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/CMakeFiles/intrinsics_gen.dir/DependInfo.cmake --color=
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/COFF /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/COFF /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/COFF/CMakeFiles/COFFOptionsTableGen.dir/DependInfo.cmake --color=
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/ELF /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/ELF /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/ELF/CMakeFiles/ELFOptionsTableGen.dir/DependInfo.cmake --color=
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/MachO /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/MachO /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/MachO/CMakeFiles/MachOOptionsTableGen.dir/DependInfo.cmake --color=
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/MinGW /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/MinGW /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/MinGW/CMakeFiles/MinGWOptionsTableGen.dir/DependInfo.cmake --color=
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f wasm/CMakeFiles/WasmOptionsTableGen.dir/build.make wasm/CMakeFiles/WasmOptionsTableGen.dir/build
/usr/bin/gmake  -f CMakeFiles/intrinsics_gen.dir/build.make CMakeFiles/intrinsics_gen.dir/build
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f COFF/CMakeFiles/COFFOptionsTableGen.dir/build.make COFF/CMakeFiles/COFFOptionsTableGen.dir/build
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f ELF/CMakeFiles/ELFOptionsTableGen.dir/build.make ELF/CMakeFiles/ELFOptionsTableGen.dir/build
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'wasm/CMakeFiles/WasmOptionsTableGen.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'COFF/CMakeFiles/COFFOptionsTableGen.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f MinGW/CMakeFiles/MinGWOptionsTableGen.dir/build.make MinGW/CMakeFiles/MinGWOptionsTableGen.dir/build
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'CMakeFiles/intrinsics_gen.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'ELF/CMakeFiles/ELFOptionsTableGen.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f MachO/CMakeFiles/MachOOptionsTableGen.dir/build.make MachO/CMakeFiles/MachOOptionsTableGen.dir/build
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'MinGW/CMakeFiles/MinGWOptionsTableGen.dir/build'.
gmake[3]: Nothing to be done for 'MachO/CMakeFiles/MachOOptionsTableGen.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
[  0%] Built target intrinsics_gen
/usr/bin/gmake  -f Common/CMakeFiles/lldCommon.dir/build.make Common/CMakeFiles/lldCommon.dir/depend
[  1%] Built target ELFOptionsTableGen
[  1%] Built target MinGWOptionsTableGen
[  2%] Built target WasmOptionsTableGen
[  2%] Built target COFFOptionsTableGen
[  3%] Built target MachOOptionsTableGen
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/Common /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/Common /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/Common/CMakeFiles/lldCommon.dir/DependInfo.cmake --color=
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f Common/CMakeFiles/lldCommon.dir/build.make Common/CMakeFiles/lldCommon.dir/build
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'Common/CMakeFiles/lldCommon.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
[ 13%] Built target lldCommon
/usr/bin/gmake  -f wasm/CMakeFiles/lldWasm.dir/build.make wasm/CMakeFiles/lldWasm.dir/depend
/usr/bin/gmake  -f COFF/CMakeFiles/lldCOFF.dir/build.make COFF/CMakeFiles/lldCOFF.dir/depend
/usr/bin/gmake  -f ELF/CMakeFiles/lldELF.dir/build.make ELF/CMakeFiles/lldELF.dir/depend
/usr/bin/gmake  -f MachO/CMakeFiles/lldMachO.dir/build.make MachO/CMakeFiles/lldMachO.dir/depend
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/COFF /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/COFF /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/COFF/CMakeFiles/lldCOFF.dir/DependInfo.cmake --color=
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/wasm /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/wasm /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/wasm/CMakeFiles/lldWasm.dir/DependInfo.cmake --color=
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/MachO /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/MachO /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/MachO/CMakeFiles/lldMachO.dir/DependInfo.cmake --color=
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/ELF /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/ELF /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/ELF/CMakeFiles/lldELF.dir/DependInfo.cmake --color=
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f COFF/CMakeFiles/lldCOFF.dir/build.make COFF/CMakeFiles/lldCOFF.dir/build
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f wasm/CMakeFiles/lldWasm.dir/build.make wasm/CMakeFiles/lldWasm.dir/build
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f MachO/CMakeFiles/lldMachO.dir/build.make MachO/CMakeFiles/lldMachO.dir/build
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'COFF/CMakeFiles/lldCOFF.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f ELF/CMakeFiles/lldELF.dir/build.make ELF/CMakeFiles/lldELF.dir/build
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'wasm/CMakeFiles/lldWasm.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
[ 26%] Built target lldWasm
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'ELF/CMakeFiles/lldELF.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
[ 42%] Built target lldCOFF
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'MachO/CMakeFiles/lldMachO.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f MinGW/CMakeFiles/lldMinGW.dir/build.make MinGW/CMakeFiles/lldMinGW.dir/depend
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/MinGW /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/MinGW /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/MinGW/CMakeFiles/lldMinGW.dir/DependInfo.cmake --color=
[ 63%] Built target lldMachO
[ 95%] Built target lldELF
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f MinGW/CMakeFiles/lldMinGW.dir/build.make MinGW/CMakeFiles/lldMinGW.dir/build
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'MinGW/CMakeFiles/lldMinGW.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
[ 97%] Built target lldMinGW
/usr/bin/gmake  -f tools/lld/CMakeFiles/lld.dir/build.make tools/lld/CMakeFiles/lld.dir/depend
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/tools/lld /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/tools/lld /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/tools/lld/CMakeFiles/lld.dir/DependInfo.cmake --color=
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f tools/lld/CMakeFiles/lld.dir/build.make tools/lld/CMakeFiles/lld.dir/build
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
gmake[3]: Nothing to be done for 'tools/lld/CMakeFiles/lld.dir/build'.
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
[ 98%] Built target lld
/usr/bin/gmake  -f test/CMakeFiles/check-lld.dir/build.make test/CMakeFiles/check-lld.dir/depend
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/test /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/CMakeFiles/check-lld.dir/DependInfo.cmake --color=
gmake[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f test/CMakeFiles/check-lld.dir/build.make test/CMakeFiles/check-lld.dir/build
gmake[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu'
[100%] Running lld test suite
cd /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test && /usr/bin/python3.8 /usr/bin/lit -sv /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test
lit: /usr/lib/python3.8/site-packages/lit/llvm/config.py:436: note: using ld.lld: /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/bin/ld.lld
lit: /usr/lib/python3.8/site-packages/lit/llvm/config.py:436: note: using lld-link: /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/bin/lld-link
lit: /usr/lib/python3.8/site-packages/lit/llvm/config.py:436: note: using ld64.lld: /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/bin/ld64.lld
lit: /usr/lib/python3.8/site-packages/lit/llvm/config.py:436: note: using wasm-ld: /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/bin/wasm-ld
lit: /usr/lib/python3.8/site-packages/lit/discovery.py:247: warning: test suite 'lld-Unit' contained no tests
FAIL: lld :: ELF/compressed-debug-level.test (674 of 2597)
******************** TEST 'lld :: ELF/compressed-debug-level.test' FAILED ********************
Script:
--
: 'RUN: at line 3';   /usr/bin/yaml2obj /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/test/ELF/compressed-debug-level.test -o /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.o
: 'RUN: at line 5';   /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/bin/ld.lld /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.o -o /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.default --compress-debug-sections=zlib
: 'RUN: at line 6';   /usr/bin/llvm-readelf --sections /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.default | /usr/bin/FileCheck -check-prefixes=HEADER,LEVEL1 /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/test/ELF/compressed-debug-level.test
: 'RUN: at line 8';   /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/bin/ld.lld -O0 /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.o -o /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.O0 --compress-debug-sections=zlib
: 'RUN: at line 9';   /usr/bin/llvm-readelf --sections /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.O0 | /usr/bin/FileCheck -check-prefixes=HEADER,LEVEL1 /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/test/ELF/compressed-debug-level.test
: 'RUN: at line 10';   cmp /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.default /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.O0
: 'RUN: at line 12';   /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/bin/ld.lld -O1 /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.o -o /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.O1 --compress-debug-sections=zlib
: 'RUN: at line 13';   /usr/bin/llvm-readelf --sections /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.O1 | /usr/bin/FileCheck -check-prefixes=HEADER,LEVEL1 /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/test/ELF/compressed-debug-level.test
: 'RUN: at line 14';   cmp /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.default /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.O1
: 'RUN: at line 16';   /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/bin/ld.lld -O2 /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.o -o /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.O2 --compress-debug-sections=zlib
: 'RUN: at line 17';   /usr/bin/llvm-readelf --sections /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/x86_64-redhat-linux-gnu/test/ELF/Output/compressed-debug-level.test.tmp.O2 | /usr/bin/FileCheck -check-prefixes=HEADER,LEVEL6 /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/test/ELF/compressed-debug-level.test
--
Exit Code: 1

Command Output (stderr):
--
ld.lld: warning: cannot find entry symbol _start; not setting start address
/home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/test/ELF/compressed-debug-level.test:23:11: error: LEVEL1: expected string not found in input
# LEVEL1: [ 1] .debug_info PROGBITS 00000000 000094 00001c
          ^
<stdin>:4:33: note: scanning from here
 [Nr] Name Type Address Off Size ES Flg Lk Inf Al
                                ^
<stdin>:6:2: note: possible intended match here
 [ 1] .debug_info PROGBITS 00000000 000094 00001b 00 C 0 0 1
 ^

Input file: <stdin>
Check file: /home/tkloczko/rpmbuild/BUILD/lld-14.0.6.src/test/ELF/compressed-debug-level.test

-dump-input=help explains the following input dump.

Input was:
<<<<<<
            1: There are 6 section headers, starting at offset 0xfc:
            2:
            3: Section Headers:
            4:  [Nr] Name Type Address Off Size ES Flg Lk Inf Al
check:23'0                                     X~~~~~~~~~~~~~~~~~ error: no match found
            5:  [ 0] NULL 00000000 000000 000000 00 0 0 0
check:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            6:  [ 1] .debug_info PROGBITS 00000000 000094 00001b 00 C 0 0 1
check:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
check:23'1      ?                                                            possible intended match
            7:  [ 2] .comment PROGBITS 00000000 0000af 000008 01 MS 0 0 1
check:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            8:  [ 3] .symtab SYMTAB 00000000 0000b8 000010 10 5 1 4
check:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            9:  [ 4] .shstrtab STRTAB 00000000 0000c8 000030 00 0 0 1
check:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           10:  [ 5] .strtab STRTAB 00000000 0000f8 000001 00 0 0 1
check:23'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           11: Key to Flags:
check:23'0     ~~~~~~~~~~~~~~
            .
            .
            .
>>>>>>

--

********************
********************
Failed Tests (1):
  lld :: ELF/compressed-debug-level.test


Testing Time: 18.61s
  Unsupported      :   14
  Passed           : 2581
  Expectedly Failed:    1
  Failed           :    1

1 warning(s) in tests
gmake[3]: *** [test/CMakeFiles/check-lld.dir/build.make:74: test/CMakeFiles/check-lld] Error 1
```

**Comment by MaskRay** [comment]
What's your zlib version? `lld/test/ELF/compressed-debug-level.test` tests the exact size of a zlib compressed content so it is somewhat brittle, but it seems work for many zlib versions from at least 1.2.7 to 1.2.11.

Also, the release/14.x likely will not receive updates and we may call this wontfix. This is a very minor issue anyway.

**Comment by kloczek** [comment]
I'm using latest zlib-ng 2.0.6.
